tools/linting/bad-urls.js

```diff
@@ -34,11 +34,11 @@ function checkURL(tree, file) {
     if (!nodeUrl) {
       return;
     }
-    const parsed = url.parse(nodeUrl);
+    const parsed = new url.URL(nodeUrl, 'https://web.dev');
```

Comment:
?

Reply:
[`url.parse` was deprecated since Node v11.0.0](https://nodejs.org/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost). (expand the history)

This effectively does the same thing, though since internal URLs don't have a base URL I'm adding one (it only get's used if the `nodeUrl` doesn't have a base URL).

Reply:
is it possible to delete the require for `url` on line 18 and change this to:

```js
const parsed = new URL(nodeUrl, site.url);
```

I think you'll need to add a require for _date/site.js